jobs/clock.js

```diff
@@ -0,0 +1,10 @@
+const cron = require('node-cron');
+const emailChallengeList = require('./email-challenge-list.js');
+
+// Every 5 minutes, from 8:00AM - 7:00 PM ETC from Monday through Friday
```

Comment:
Changed. It runs from 12 - 23 UTC everyday. Which is the equivalent of 8:00AM - 7:00PM EST